Add thresholds to CPA analyzers. (#43)
MicrosoftDocs/mesh-docs: mesh/develop/debug-and-optimize-performance/cpa.md

## Analyzers

> Each analyzer performs a different test to check if content is optimally setup. The below table highlights what each analyzer does as well as discrete thresholds.

Reviewer: ```suggestion
Each analyzer performs a different test to check if content is optimally setup. The table below highlights what each analyzer does as well as discrete thresholds.
```

| Name | Description | Requires Play Mode | PC thresholds | Android (Quest) thresholds |
| -------- | ------- | ------- | ------- | ------- |

> | **Animation** | Checks if Animator and Animation components have visibility culling setup optimally. | No | N/A | N/A |

Reviewer: ```suggestion
| **Animation** | Checks if Animator and Animation components have visibility culling set up optimally. | No | N/A | N/A |
```

> | **Collision Complexity** | Checks for high triangle count MeshCollider components. | No | Under 100 triangles for dynamic MeshColliders and under 10,000 triangles for static MeshColliders | Same as PC |

Author: ```suggestion
| **Collision Complexity** | Checks for high triangle count MeshCollider components. | No | Under 100 triangles for dynamic MeshColliders and under 10,000 triangles for static MeshColliders. | Same as PC |
```

> | **Light Complexity** | Checks utilization of realtime lights and shadow casting lights. And for the existence of an avatar light. | No | Under 5 realtime per pixel lights. And 0 realtime shadow casting lights. | Under 3 realtime per pixel lights. And 0 realtime shadow casting lights. |

Reviewer: ```suggestion
| **Light Complexity** | Checks utilization of real-time lights and shadow casting lights and for the existence of an avatar light. | No | Under 5 real-time per pixel lights and 0 real-time shadow casting lights. | Under 3 real-time per pixel lights and 0 real-time shadow casting lights. |
```

| **Mesh Complexity** | Warns about the use of very dense MeshFilter components. | No | Above 30,000 triangles per MeshFilter emits a warning and above 100,000 triangles per MeshFilter emits an error. | Same as PC |
| **Mesh Complexity Rendered** | Checks if a viewpoint is rendering too many triangles. | Yes | Under 500,000 triangles rendered. | Under 80,000 triangles rendered. |

> | **Reflection Probes** | Checks if realtime reflection probes are being used. | No | Allowed | Disallowed |

Reviewer: ```suggestion
| **Reflection Probes** | Checks if real-time reflection probes are being used. | No | Allowed | Disallowed |
```

> | **Rigidbody Complexity** | Checks for too many Rigidbody components being used. | No | Under 50 rigid bodies. | Same as PC |

Reviewer: ### 
```suggestion
| **Rigidbody Complexity** | Checks for too many Rigidbody components being used. | No | Under 50 Rigidbodies. | Same as PC |
```

> | **Scene Complexity** | Checks if a viewpoint is emitting too many rendering batches. | Yes | Under 200 batches emitted. | Under 50 batches emitted |

Author: ```suggestion
| **Scene Complexity** | Checks if a viewpoint is emitting too many rendering batches. | Yes | Under 200 batches emitted. | Under 50 batches emitted. |
```

> | **Shader Complexity** | Checks the number vertex and fragment stage math operations of referenced shaders. | No | Under 150 math operations for the vertex stage. Under 600 math operations for the fragment stage. | Under 30 math operations for the vertex stage. Under 120 math operations for the fragment stage. |

Reviewer: ```suggestion
| **Shader Complexity** | Checks the number of vertex and fragment stage math operations of referenced shaders. | No | Under 150 math operations for the vertex stage. Under 600 math operations for the fragment stage. | Under 30 math operations for the vertex stage. Under 120 math operations for the fragment stage. |
```

| **SRP Batcher Compatible** | Checks if SRP Batcher compatible shaders are being used. | No | N/A | N/A |

> | **Terrain** | Checks for performance issues on Terrain components. | No | Under 2 Terrain components and a Heightmap Pixel Error over 4 | Same as PC |

Author: ```suggestion
| **Terrain** | Checks for performance issues on Terrain components. | No | Under 2 Terrain components and a Heightmap Pixel Error over 4. | Same as PC |
```
